doc/data_protocol.md

```diff
@@ -126,9 +183,11 @@ The source file can be found in [doc/diagrams/command-sequence.xml](diagrams/com
 
 ### Commands description
 
+ALL DATA MUST BE LITTLE-ENDIAN
+
 #### Engine Controller
 
-Commands from the Ground Station to the Engine Computer
+Commands sent from the Ground Station to the Engine Computer
 
```

Comment:
I'd suggest removing the table until the protocol has been better defined & tested, but we could add the following:
```suggestion
Commands sent from the Ground Station to the Engine Computer
- Set power mode
- Arming high power outputs
- Power/depower high power outputs
- Initiate controlled engine ignition (only possible if all outputs are armed and other criteria are met)
- Abort controlled engine ignition (might not be possible)
- Time sync responses
- ... Other messages
```

Reply:
See previous answer.